packages/Tag/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@paprika/tag",
+  "version": "1.0.0",
```

Comment:
I think we want to bump this to `v1.0.0` during the first publish, so maybe drop this to `0.0.1`?

Reply:
So why shouldn't I leave it as 1.0.0 then? As it hasn't been published yet?

Reply:
Not a big deal, but when it gets published it will get incremented to `v1.0.1`.

Reply:
Ok, i wasn't aware, good to know, i'll set as 0.0.1 then for now